src/getCoinCombination.test.js

```diff
@@ -1,10 +1,33 @@
 'use strict';
 
+
 describe('getCoinCombination', () => {
-  const { getCoinCombination } = require('./getCoinCombination');
```

Comment:
The function 'getCoinCombination' is not imported. Make sure to import this function before invoking it.